server/store/vehicles.js

```diff
@@ -31,18 +31,21 @@ const addNewVehicle = vehicle => {
   updateVehiclePosition(vehicle);
 
   // Set TTL for vehicles
-  redis.expire(`vehicles:${vehicle.id}`, config('vehicles_ttl'));
+  setVehicleTTL(vehicle.id);
   // Send new vehicle to Captain
   createVehicle(vehicle);
 };
 
 const getVehicle = async id => {
-  redis.expire(`vehicles:${id}`, config('vehicles_ttl'));
+  setVehicleTTL(id);
   let vehicle = await redis.hgetallAsync(`vehicles:${id}`);
   vehicle.coords = {long: vehicle.long, lat: vehicle.lat};
   return vehicle;
 };
 
+const setVehicleTTL = async vehicleId => 
```

Comment:
As @TalAter mentioned in a similar pr, https://github.com/DAVFoundation/missioncontrol/pull/159#pullrequestreview-97387026

> I think you can remove the `async` from this line. We don't actually have to await for a response from this one.

Reply:
I totally agree with you.. Done !